scene/2d/audio_stream_player_2d.cpp

```diff
@@ -140,6 +140,7 @@ void AudioStreamPlayer2D::_update_panning() {
 
 		//screen in global is used for attenuation
 		AudioListener2D *listener = vp->get_audio_listener_2d();
+		ERR_FAIL_NULL(listener);
```

Comment:
How does it fix a crash? The use of `listener` before is guarded by `if (listener)` (which is now made redundant so the `else` branch would be dead code.

Reply:
Huh, that's a good question. It does fix the crash though, I posted stack trace in the issue. But it doesn't make sense .-.